src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.lock.json

```diff
@@ -4,10 +4,78 @@
   "targets": {
     "DNXCore,Version=v5.0": {
       "coveralls.io/1.4": {},
-      "Microsoft.DotNet.PerfTools/0.0.1-prerelease-00077": {},
-      "Microsoft.NETCore.Runtime/1.0.1-beta-23213": {},
-      "Microsoft.NETCore.TestHost-x64/1.0.0-beta-23213": {},
-      "Microsoft.NETCore.TestHost-x86/1.0.0-beta-23213": {},
+      "Microsoft.DotNet.xunit.performance.analysis/1.0.0-alpha-build0012": {},
+      "Microsoft.DotNet.xunit.performance.runner.Windows/1.0.0-alpha-build0012": {
+        "dependencies": {
+          "xunit.runner.console": "2.1.0-beta4-build3109"
```

Comment:
I'm actually not sure where this dependency is being pulled in from, as I did update the nuspec for the runner to point at beta3. Perhaps there was some issue with my NuGet cache or something. I'll check this out tomorrow.